plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeConnectorSmokeTest.java

```diff
@@ -407,20 +411,20 @@ private void testDropTable(String tableName, String resourcePath)
     @Test
     public void testDropAndRecreateTable()
     {
-        String tableName = "testDropAndRecreate_" + randomTableSuffix();
-        assertUpdate(format("CREATE TABLE %s (dummy int) WITH (location = '%s')", tableName, getLocationForTable(bucketName, "nation")));
+        String tableName = "test_drop_and_recreate_" + randomTableSuffix();
```

Comment:
I would recommend extracting a commit as this is unrelated change.

Reply:
Addressed!